sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentField.cs

```diff
@@ -77,10 +77,19 @@ internal DocumentField(DocumentFieldType fieldType, DocumentFieldValue value, st
         }
 
         /// <summary>
-        /// The data type of the field value.
+        /// The data type of the field value. If <see cref="DocumentFieldType.Unknown"/>,
+        /// it means the value of the field could not be parsed by the service. The expected
+        /// field type can be checked at <see cref="ExpectedFieldType"/>. Consider using
+        /// <see cref="Content"/> to get a textual representation of the field and parsing it
+        /// manually in this case.
+        /// </summary>
+        public DocumentFieldType FieldType => Value.FieldType;
```

Comment:
Should this be ActualFieldType perhaps? One looks at ExpectedFieldType and FieldType and it is confusing what FieldType represents if ExpectedFieldType is also available in the public API surface area? Just a thought - if this has been arch board reviewed already - I wouldn't block on it.

Reply:
We added the `ExpectedFieldType` property to handle a very particular scenario, but most users will never need to use it. One of the things we had in mind when coming up with this design was keeping the usage simple for the average user. I think that having `Actual`/`Expected` could make it harder for users to know which one to use (on the other hand, it would force them to think better about it).

That said, these names were suggested by architects, so I also wanted to avoid additional changes.